### PR TITLE
Add docs for `Dir#path`

### PR DESCRIPTION
**Pull request by wontruefree**

### CODE REVIEW

File: src/dir.cr [hunk 1]
```diff
@@ -11,6 +11,17 @@ class Dir
   include Enumerable(String)
   include Iterable(String)
 
+  # Returns the path of Dir from the location from the location initialized.
```
Comment (straight-shoota) [inline src/dir.cr:14]:
```suggestion
  # Returns the path of this directory.
```

**Review by straight-shoota — APPROVED** [review]

File: src/dir.cr [hunk 2]
```diff
@@ -11,6 +11,17 @@ class Dir
...
+  # Returns the path of this directory.
+  #
+  # ```
+  # Dir.mkdir("testdir")
+  # dir = Dir.new("testdir")
+  # Dir.mkdir("testdir/extendeddir")
+  # dir2 = Dir.new("testdir/extendeddir")
+  #
+  # dir.path  # => "testdir"
+  # dir2.path # => "testdir/extendeddir"
```
[the remark this answers is not in the document]
Comment (wontruefree) [reply src/dir.cr:23]: I saw your other suggestion to rename dirs to foo and bar.  There are other examples with `testdir`.  I am not sure using `foo` and `bar` are good for these examples.

**Review by beta-ziliani — APPROVED** [review]
Thanks @wontruefree !

**Comment by wontruefree** [comment]
@beta-ziliani you are welcome I am learning a lot about the std lib. I thought I understood it well but I am learning a lot more.

**Comment by beta-ziliani** [comment]
I will use you as an example of "learning by contributing" :-) (I mean it, BTW)

**Pull request merged**